docs/_sidebar.md

```diff
@@ -6,6 +6,7 @@
 - [Edit TechPass profile](edit-profile)
 - [Onboard to SEED](onboard-to-seed)
 - [Reset MFA](reset-mfa)
+- [System Status](status)
```

Comment:
same for this option. let's remove it from the options until there's real data.
I will suggest to put in the prod maintenance details if we really want to publish this page

Reply:
noted